google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/BreakpointUtil.java

```diff
@@ -84,11 +86,17 @@ public static Date parseDateTime(@Nullable String dateString) {
     dateString = dateString.replaceAll("Z$", "-0000");
 
     SimpleDateFormat iso8601Format = new SimpleDateFormat(ISO_8601_FORMAT);
+    iso8601Format.setLenient(true);
 
     try {
```

Comment:
Might be overkill but what do you think about refactoring to something like this:

```
public static final String supportedFormats = Arrays.asList(ISO_.., ISO..);
..
for (String formatString : formats) {
  try {
   ...
   return format.parse(dateString);
  } catch (..) {
   LOG.error("Failed to parse datetime %s using format %s", ..);
  }
}
```